### ISSUE TITLE
Integration not loading on latest HA Code 2024.12.0

### ISSUE THREAD

**Issue by sayam93** [arevindh/tinxylocal#5]
The tinxylocal custom integration stopped working for me after upgrading HA Core to 2024.12.0

Error Log below:

```
Logger: homeassistant.setup
Source: setup.py:334
First occurred: 1:03:47 AM (1 occurrences)
Last logged: 1:03:47 AM

Setup failed for custom integration 'tinxylocal': Unable to import component: Error relocating /config/custom_components/tinxylocal/pyarmor_runtime_000000/linux_aarch64/pyarmor_runtime.so: _PyErr_GetTopmostException: symbol not found
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1014, in async_get_component
    comp = await self.hass.async_add_import_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self._get_component, True
        ^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/local/lib/python3.13/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1074, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/tinxylocal/__init__.py", line 13, in <module>
    from .coordinator import TinxyUpdateCoordinator
  File "/config/custom_components/tinxylocal/coordinator.py", line 12, in <module>
    from .hub import TinxyConnectionException, TinxyLocalException, TinxyLocalHub
  File "/config/custom_components/tinxylocal/hub.py", line 10, in <module>
    from .encrypt import PasswordEncryptor
  File "/config/custom_components/tinxylocal/encrypt.py", line 2, in <module>
    from .pyarmor_runtime_000000 import __pyarmor__
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 53, in <module>
    __pyarmor__ = __pyarmor__().__pyarmor__
                  ~~~~~~~~~~~^^
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 52, in __pyarmor__
    return __import__(name, globals(), locals(), ['__pyarmor__'], level=1)
ImportError: Error relocating /config/custom_components/tinxylocal/pyarmor_runtime_000000/linux_aarch64/pyarmor_runtime.so: _PyErr_GetTopmostException: symbol not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 334, in _async_setup_component
    component = await integration.async_get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1034, in async_get_component
    self._component_future.result()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1026, in async_get_component
    comp = self._get_component()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1074, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/tinxylocal/__init__.py", line 13, in <module>
    from .coordinator import TinxyUpdateCoordinator
  File "/config/custom_components/tinxylocal/coordinator.py", line 12, in <module>
    from .hub import TinxyConnectionException, TinxyLocalException, TinxyLocalHub
  File "/config/custom_components/tinxylocal/hub.py", line 10, in <module>
    from .encrypt import PasswordEncryptor
  File "/config/custom_components/tinxylocal/encrypt.py", line 2, in <module>
    from .pyarmor_runtime_000000 import __pyarmor__
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 53, in <module>
    __pyarmor__ = __pyarmor__().__pyarmor__
                  ~~~~~~~~~~~^^
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 52, in __pyarmor__
    return __import__(name, globals(), locals(), ['__pyarmor__'], level=1)
ImportError: Error relocating /config/custom_components/tinxylocal/pyarmor_runtime_000000/linux_aarch64/pyarmor_runtime.so: _PyErr_GetTopmostException: symbol not found
```

**Comment by sayam93** [comment]
More logs

```
2024-12-05 01:04:26.828 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module with args ('custom_components.tinxylocal.config_flow',) in /usr/src/homeassistant/homeassistant/loader.py, line 1303: return importlib.import_module(f"{self.pkg_path}.{platform_name}") inside the event loop; This is causing stability issues. Please create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue
For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#import_module
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 227, in <module>
    sys.exit(main())
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 213, in main
    exit_code = runner.run(runtime_conf)
  File "/usr/src/homeassistant/homeassistant/runner.py", line 154, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.13/asyncio/base_events.py", line 708, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.13/asyncio/base_events.py", line 679, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.13/asyncio/base_events.py", line 2027, in _run_once
    handle._run()
  File "/usr/local/lib/python3.13/asyncio/events.py", line 89, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 112, in sem_task
    return await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1315, in async_init
    flow, result = await self._async_init(flow_id, handler, context, data)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1340, in _async_init
    flow = await self.async_create_flow(handler, context=context, data=data)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1536, in async_create_flow
    handler = await _async_get_flow_handler(
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 3333, in _async_get_flow_handler
    await _load_integration(hass, domain, hass_config)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 3312, in _load_integration
    await integration.async_get_platform("config_flow")
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1107, in async_get_platform
    platforms = await self.async_get_platforms((platform_name,))
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1172, in async_get_platforms
    platforms.update(self._load_platforms(platform_names))
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1097, in _load_platforms
    platform_name: self._load_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1271, in _load_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1303, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
2024-12-05 01:04:26.981 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration tinxylocal: Error relocating /config/custom_components/tinxylocal/pyarmor_runtime_000000/linux_aarch64/pyarmor_runtime.so: _PyErr_GetTopmostException: symbol not found
2024-12-05 01:04:27.051 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 3312, in _load_integration
    await integration.async_get_platform("config_flow")
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1107, in async_get_platform
    platforms = await self.async_get_platforms((platform_name,))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1184, in async_get_platforms
    import_future.result()
    ~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1172, in async_get_platforms
    platforms.update(self._load_platforms(platform_names))
                     ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1097, in _load_platforms
    platform_name: self._load_platform(platform_name)
                   ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1271, in _load_platform
    cache[full_name] = self._import_platform(platform_name)
                       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1303, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/tinxylocal/__init__.py", line 13, in <module>
    from .coordinator import TinxyUpdateCoordinator
  File "/config/custom_components/tinxylocal/coordinator.py", line 12, in <module>
    from .hub import TinxyConnectionException, TinxyLocalException, TinxyLocalHub
  File "/config/custom_components/tinxylocal/hub.py", line 10, in <module>
    from .encrypt import PasswordEncryptor
  File "/config/custom_components/tinxylocal/encrypt.py", line 2, in <module>
    from .pyarmor_runtime_000000 import __pyarmor__
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 53, in <module>
    __pyarmor__ = __pyarmor__().__pyarmor__
                  ~~~~~~~~~~~^^
  File "/config/custom_components/tinxylocal/pyarmor_runtime_000000/__init__.py", line 52, in __pyarmor__
    return __import__(name, globals(), locals(), ['__pyarmor__'], level=1)
ImportError: Error relocating /config/custom_components/tinxylocal/pyarmor_runtime_000000/linux_aarch64/pyarmor_runtime.so: _PyErr_GetTopmostException: symbol not found
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/discovery_flow.py", line 137, in _async_start
    await gather_with_limited_concurrency(FLOW_INIT_LIMIT, *init_coros)
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 114, in gather_with_limited_concurrency
    return await gather(
           ^^^^^^^^^^^^^
    ...<2 lines>...
    )
    ^
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 112, in sem_task
    return await task
           ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1315, in async_init
    flow, result = await self._async_init(flow_id, handler, context, data)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1340, in _async_init
    flow = await self.async_create_flow(handler, context=context, data=data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1536, in async_create_flow
    handler = await _async_get_flow_handler(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.hass, handler_key, self._hass_config
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 3333, in _async_get_flow_handler
    await _load_integration(hass, domain, hass_config)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 3319, in _load_integration
    raise data_entry_flow.UnknownHandler from err
homeassistant.data_entry_flow.UnknownHandler
```

**Comment by sayam93** [comment]
Fixed with v1.1.0
